backend/hitas/oracle_migration/oracle_schema/company.py

```diff
@@ -12,13 +15,13 @@
     "HITYHTIO",
     metadata_obj,
     Column("KG_YTUNNUS", Integer, key="id", primary_key=True),
-    Column("C_YHTNIMI", String(100), key="official_name", nullable=False),
-    Column("C_HAKUNIMI", String(50), key="display_name", nullable=False),
-    Column("C_KATUOS", String(50), key="address", nullable=False),
+    Column("C_YHTNIMI", HitasAnonymizedAddress(100, unique=True), key="official_name", nullable=False),
```

Comment:
This is fine, but maybe later we can use real data or anonymise the field based on some argument 🤔